geometric_features/aggregation/__init__.py

```diff
@@ -52,6 +53,9 @@ def get_aggregator_by_name(region_group):
                'ISMIP6 Regions': {'prefix': 'ismip6Regions',
                                   'date': '20210201',
                                   'function': ismip6},
+               'MOC Basins': {'prefix': 'mocBasins',
+                              'date': '20210331',
```

Comment:
Mask files without the southern transects will be named `<mesh>_mocBasins20210331.nc`.  Then, the ones with the southern transect (calculated via MPAS-Tools) will be `<mesh>_mocBasinsAndTransects20210331.nc`.  Does that seem right?

Reply:
I wonder if we need both? just `_mocBasinsAndTransects` would be enough in my view, since these masks are only used in the MOC and MHT am's?

Reply:
As a practical consideration, at the moment I don't have a good way of checking if `<mesh>_mocBasinsAndTransects20210331.nc` and not creating `<mesh>_mocBasins20210331.nc` if it does.  I'll have to think about how to do this without making things unnecessarily complicated.

Reply:
Never mind, that's quite easy to handle.